avogadro/qtplugins/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 find_package(Eigen3 REQUIRED)
 include_directories(SYSTEM ${EIGEN3_INCLUDE_DIR})
 
-find_package(Qt5 COMPONENTS Widgets OpenGL Network Concurrent REQUIRED)
+find_package(Qt5 COMPONENTS Widgets OpenGL Network Concurrent WebView REQUIRED)
```

Comment:
I don't think this is needed, and the others should be removed. There was a bug I was working around, we can leave this in for now, and I will fix all of them.